### PR TITLE
Nps tilda operator

### PR DESCRIPTION
**Pull request by jancervenka**

### CODE REVIEW

**Comment by jancervenka** [comment]
I have added tests for the new tilda operator and refactored `tests/toolking/test_parser.py` (I removed the `unittest.TestCase`) and changed test imports `from epstats.foo.bar import x` –> `from src.epstats.foo.bar import x`. This prevents the situation when the tests are using `epstats` package installed in your environment instead of the `epstats` in the repository.

File: tests/epstats/server/test_api_evaluate.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
 import pandas as pd
 from fastapi.testclient import TestClient
 
-from epstats.toolkit.testing import (
```
Comment (ondraz) [inline tests/epstats/server/test_api_evaluate.py:4]: It should all work without the need to start all imports with `src`. Running of `make install-dev` should install the package with `pip install -e .` to the python env. So we do not need to start imports with `src`. Repeats many times in changed files.

**Review by ondraz — APPROVED** [review]
LGTM, please fix that `src` in imports. Thanks a lot!

File: tests/epstats/server/test_api_evaluate.py (hunk 1 above)
Comment (ondraz) [reply tests/epstats/server/test_api_evaluate.py:4]: Ahh, ok, I got it. This `src` is just for files with tests. It's ok then and thanks for fixing it!

**Review by samuelpucek — APPROVED** [review]
LGTM!

**Pull request merged**